build-system/tasks/report-test-status.js

```diff
@@ -1,236 +0,0 @@
-/**
- * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS-IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-'use strict';
-
-const argv = require('minimist')(process.argv.slice(2));
-const fetch = require('node-fetch');
-const {
-  isCircleciBuild,
-  isGithubActionsBuild,
-  isPullRequestBuild,
-} = require('../common/ci');
-const {ciJobUrl} = require('../common/ci');
-const {cyan, yellow} = require('../common/colors');
-const {getValidExperiments} = require('../common/utils');
```

Comment:
Looks like this is the only spot where `getValidExperiments` was being used. If so, it too can be deleted from  `utils.js`.

Reply:
Excellent catch